packages/table/src/regions.ts

```diff
@@ -548,11 +541,52 @@ export class Regions {
         }
     }
 
+    private static overlapsRegionInternal(regions: IRegion[], query: IRegion, allowPartialOverlap = true) {
```

Comment:
do you know about `/** @internal */` JSDoc tag? things marked suchly will be stripped from the `.d.ts` typings.